typescript/infra/config/warp.ts

```diff
@@ -1,73 +1,35 @@
-import { ethers } from 'ethers';
-
 import {
   ChainMap,
-  HyperlaneIsmFactory,
   MultiProvider,
+  RouterConfig,
   TokenRouterConfig,
-  TokenType,
-  buildAggregationIsmConfigs,
-  defaultMultisigConfigs,
 } from '@hyperlane-xyz/sdk';
 
-import { Modules, getAddresses } from '../scripts/agent-utils.js';
 import { getHyperlaneCore } from '../scripts/core-utils.js';
 import { EnvironmentConfig } from '../src/config/environment.js';
-import { tokens } from '../src/config/warp.js';
 
-import { DEPLOYER } from './environments/mainnet3/owners.js';
+import { getAncient8EthereumUSDCWarpConfig } from './environments/mainnet3/warp/getAncient8EthereumUSDCWrapConfig.js';
+
+export const warpConfigGetterMap: Record<
```

Comment:
Renzo deployments that add chains will mean that this will need to be updated

Reply:
yeah :/
did we ever flag this to Rossy / the product team that the warp ID pattern makes cases like this really inconvenient? otherwise may be good to